understanding/20/images-of-text.html

```diff
@@ -130,139 +120,54 @@ <h2>Examples of Images of Text</h2>
          <dt>Customizable font settings in images of text</dt>
          <dd>A Web site allows users to specify font settings and all images of text on the site
                are then provided based on those settings.</dd>
+         <dt>The text replicates an image of text</dt>
```

Comment:
> The text replicates an image of text

I have read "replicates" the same way as in the inverse situation of "image as text" where the image replactes the text. The text below than makes clear the content passes becasue equivalent text is rendered below. The heading does not say this clearly, though.